docs/content/connectors/oracle-cdc.md

```diff
@@ -305,6 +305,11 @@ CREATE TABLE products (
 );
 ```
 
+Tips:
+
+1. Oracle is case-sensitive and by default it will convert field names to uppercase if they are not quoted. However, Flink SQL will not do it. So for physical columns, we have to exactly match the case as defined in Oracle. Otherwise, we will get null values.
+2. For meta columns both uppercase and lowercase are supported.
+
```

Comment:
How about this ? ```suggestion
** Note ** : The Oracle dialect is case-sensitive, it converts field name to uppercase if the field name is not quoted, Flink SQL doesn't convert the field name. Thus for physical columns from oracle database, we should use its converted field name in Oracle when define an `oracle-cdc` table in Flink SQL. 
```

Reply:
Thanks. It looks good to me

Reply:
There should be a blank line before Chapter Features.